gallery/lib/pages/splash.dart

```diff
@@ -85,21 +83,10 @@ class _SplashPageState extends State<SplashPage>
         duration: Duration(
           milliseconds: splashPageAnimationDurationInMilliseconds,
         ),
-        value: 1,
```

Comment:
Should we not remove this duration here? As it otherwise would wait before showing the rest of the page.

There's also other changes related to `splashPageAnimationDurationInMilliseconds` that can should be removed if we remove the splash animation.